### ISSUE TITLE
[4.0.0-beta.4] GQL isn't working?

### ISSUE THREAD

**Issue by davidhellmann** [craftcms/ckeditor#163]
### Description
Tried to use GQL within the GQL Playground but looks like it isn't working. Tested also with Craft 4.x and CKE 3.x and this is working.

### Steps to reproduce
![image](https://github.com/craftcms/ckeditor/assets/764292/5a3e1ef7-6b6b-4d80-91d8-652b29a217f1)
![image](https://github.com/craftcms/ckeditor/assets/764292/bb3c1dd1-1a45-48f3-94ad-dbf7a2bab22b)

### Additional info

- Craft version: 5.beta-1

**Comment by davidwebca** [comment]
Just realized the same thing just now. I tried changing the configs, creating new fields, with or without sub entries inside the field. Just came here to mention this in case it helps!

**Comment by davidwebca** [comment]
I take it back, I'm actually backtracking to fix the issue by downgrading the plugin one version at a time and I now realize that since v4 beta 1, it's been failing but I hadn't realized because they were released in quick successions.

**Comment by brandonkelly** [comment]
Thanks for reporting that! Just released CKEditor 4.0.0-beta.5 with a fix.

**Comment by davidhellmann** [comment]
Follow-up question @brandonkelly 

The output with entry blocks looks like this:

```JSON file
"textEditorDefault": "<p>Quo cu eius tollit instructior.</p><p>Entry 4051</p><p>Quo cu eius tollit instructior.</p><p>Entry 4052</p><p>An populo iriure interesset sit!.</p>"
},
```

Is a children's method coming, or how does that work in the future? Just want to know if I can build on top of that or go straight with a Matrix field at the moment. 

Thank you!

**Comment by carlcs** [comment]
@davidhellmann You need to set up element partials for the entry types that you're embedding within your CKEditor fields.

https://craftcms.com/docs/5.x/system/elements.html#rendering-elements

**Comment by davidhellmann** [comment]
Oh, thought that was also for twig templates. I'll have a look. Thank you!

**Comment by brandonkelly** [comment]
Just released Craft 5.0.0-beta.2 and CKEditor 4.0.0-beta.6, which adds support for querying and mutating nested entries within CKEditor fields directly (https://github.com/craftcms/cms/pull/14366).

So if you want to fetch nested entry data without having it pre-rendered as HTML via the partial template, you could have your partial template just include a placeholder referencing the entry ID, e.g. 

```twig
<div data-craft-entry-id="{{ entry.id }}"></div>
```

And then fetch the nested entry data via a second GraphQL query, and do whatever you want from there.

Note that if you’re creating _new_ nested entries via GraphQL, you will need to update the parent element’s CKEditor field value with a new `<craft-entry>` tag referencing it, before it will actually show up within the field.

```html
<craft-entry data-entry-id="123"></craft-entry>
